arch/lkl/include/asm/uaccess.h

```diff
@@ -0,0 +1,64 @@
+#ifndef _ASM_LKL_UACCESS_H
+#define _ASM_LKL_UACCESS_H
+
+/* copied from old include/asm-generic/uaccess.h */
```

Comment:
Why can't we reuse the ones from asm-generic?

Reply:
this part is not copy of *current* asm-generic/uaccess.h, but from the old one (last LKL based version).

The commit d597580d373774b1bdab84b3d26ff0b55162b916 introduces raw_copy_from_user & co, which refactors a lot and require each-arch implementation of raw_copy_from_user.  I dont have why but this commit is fix to the above update.